api/src/geolocate/get.handler.ts

```diff
@@ -0,0 +1,95 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+import * as AWS from 'aws-sdk';
+import { RDSDataService } from 'aws-sdk';
+import {
+  ExecuteStatementRequest,
+  FieldList,
+  SqlParametersList,
+} from 'aws-sdk/clients/rdsdataservice';
+
+const SCHEMA = 'public';
+
+/**
+ * Returns all geo identifiers that intersect with a lat, long.
+ * @param rdsService: RDS service to connect to the db.
+ * @param params: with lat and long coordinates sued to query tables.
+ * @param geoid: identifier to retrieve for a table.
+ * @param table: db table to query.
+ */
+async function getGeoDataForLatLong(
+  rdsService: RDSDataService,
+  params: SqlParametersList,
+  geoid: string,
+  table: string,
+): Promise<string | undefined> {
+  const executeParams: ExecuteStatementRequest = {
+    database: process.env.DATABASE_NAME ?? 'postgres',
+    resourceArn: process.env.RESOURCE_ARN ?? '',
+    schema: SCHEMA,
+    secretArn: process.env.CREDENTIALS_SECRET ?? '',
+    sql: `SELECT ${geoid}
+          FROM ${table}
+          WHERE geom && ST_SetSRID(ST_Point(:long, :lat), 4326)
```

Comment:
This might need to use [`ST_Contains`](https://postgis.net/docs/ST_Contains.html) to check that the point is within `geom` ([possibly helpful example](https://gis.stackexchange.com/a/83555).

Reply:
Added a todo for sake of getting this in

Reply:
It's intersects in postgis https://postgis.net/docs/geometry_overlaps.html

Reply:
Removed the todo as well